tests/.wickerconfig.test.json

```diff
@@ -1,10 +1,19 @@
 {
     "aws_s3_config": {
       "s3_datasets_path": "s3://SOME_FAKE_PATH",
-      "region": "us-west-2"
+      "region": "us-west-2",
+      "boto_config": {
+        "max_pool_connections":10,
+        "read_timeout": 140,
```

Comment:
nit: specify unit here and elsewhere: `read_timeout_s`

Reply:
Lint issue. Corrected.